### PR TITLE
function duplicateRows added

### PR DESCRIPTION
**Pull request by cbeltrangomez84**
I added a function to duplicate rows with style and values in a worksheet. It still needs to be improve since formulas are not copied and it does not support merged cells, this function was created to support issue #839

### CODE REVIEW

**Comment by Siemienik** [comment]
@cbeltrangomez84  something went wrong during testing

**Comment by cbeltrangomez84** [comment]
Sorry @Siemienik I don´t know how to do it. 😳
I sent you an email.

**Comment by Siemienik** [comment]
@cbeltrangomez84 ok, each important thinks has been done:),

If I ask You, please write some documentation to README how to use this function :)

File: lib/doc/worksheet.js [hunk 1]
```diff
@@ -344,6 +344,49 @@ class Worksheet {
...
+  duplicateRow(start, count) {
+    // I want to add after the start row
+    start++;
+
+    const nKeep = start;
+    const nEnd = this._rows.length;
+    let i;
+    let rSrc;
+
+    // insert new cells
```
Comment (guyonroche) [inline lib/doc/worksheet.js]: Thinking out loud here - this for loop is a duplication of code already implemented in worksheet.spliceRows()
Perhaps a better strategy would be to prepare an array of insertion values and let spliceRows() do the donkey work...
```javascript
const values = this._rows[start - 1].values;
const inserts = new Array(count - 1).fill(values);
const deleteCount = insert ? 0 : count - 1;
this.spliceRows(start + 1, deleteCount, ...inserts);

// copy styles from start row to cloned rows
...
```

**Comment by cbeltrangomez84** [comment]
Thanks @guyonroche and @Siemienik 

I made another pull request with the changes and README (plus another test)

**Pull request merged**